viz/cmd/stat.go

```diff
@@ -188,7 +188,7 @@ If no resource name is specified, displays stats about all resources of the spec
   linkerd viz stat serverauthorization/web-public
 
   # Get all inbound stats to the web-get and web-delete HTTP route resources
-  linkerd viz stat route/web-get route/web-delete
+  linkerd viz stat httproute/web-get httproute/web-delete
```

Comment:
`route` in this example refers to a ServiceProfile route rather than an `HttpRoute` resource.  This ambiguity between ServiceProfile routes and `HttpRoute` is something we plan to clarify in future releases, but for now this example should be left unchanged.